server/infra/database/GrowerAccountRepository.ts

```diff
@@ -33,6 +33,20 @@ export default class GrowerAccountRepository extends BaseRepository<GrowerAccoun
       result.whereIn(whereIn.field, whereIn.values);
     });
 
+    Object.entries(parameters).forEach(([key, val]) => {
+      if (val === 'not null') {
+        result.whereNotNull(key);
+        delete parameters[key];
+      }
+    });
+
+    Object.entries(parameters).forEach(([key, val]) => {
+      if (val === 'null') {
```

Comment:
Also we could merge the two forEach(s)

if(val === 'null'){
} else if (val === 'not null') {
}